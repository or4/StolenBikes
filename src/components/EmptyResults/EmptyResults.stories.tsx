import * as React from 'react';
import { storiesOf } from '@storybook/react';

import { EmptyResults } from './EmptyResults';

const stories = storiesOf('EmptyResults', module);

stories.add('EmptyResults', () => {
    return <EmptyResults />;
});
