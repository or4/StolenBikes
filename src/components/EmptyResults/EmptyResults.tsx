import React from 'react';
import styled from 'styled-components';

const Container = styled.div``;

export const EmptyResults: React.FC = () => <Container>No results</Container>;
