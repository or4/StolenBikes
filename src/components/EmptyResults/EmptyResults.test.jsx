import React from 'react';
import { mount } from 'enzyme';

import { EmptyResults } from './EmptyResults';

describe('EmptyResults', () => {
    it('should render correctly', () => {
        const component = <EmptyResults />;

        const container = mount(component);
        expect(container).toMatchSnapshot();
        container.unmount();
    });
});
