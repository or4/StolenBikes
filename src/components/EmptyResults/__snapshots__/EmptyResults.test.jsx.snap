// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EmptyResults should render correctly 1`] = `
<EmptyResults>
  <styled.div>
    <StyledComponent
      forwardedComponent={
        Object {
          "$$typeof": Symbol(react.forward_ref),
          "attrs": Array [],
          "componentStyle": ComponentStyle {
            "componentId": "sc-bdVaJa",
            "isStatic": true,
            "lastClassName": "bDWFJH",
            "rules": Array [],
          },
          "displayName": "styled.div",
          "foldedComponentIds": Array [],
          "render": [Function],
          "styledComponentId": "sc-bdVaJa",
          "target": "div",
          "toString": [Function],
          "warnTooManyClasses": [Function],
          "withComponent": [Function],
        }
      }
      forwardedRef={null}
    >
      <div
        className="sc-bdVaJa bDWFJH"
      >
        No results
      </div>
    </StyledComponent>
  </styled.div>
</EmptyResults>
`;
